content/docs/rfcs/16/README.md

```diff
@@ -129,7 +130,7 @@ The following structured types are defined for use on the Relay API:
 | `payload` | `String` | mandatory | The payload being relayed as a hex encoded data string |
 | `contentTopic` | `String` | optional | Message content topic for optional content-based filtering |
 | `timestamp` | `Number` | optional | The time at which the message is generated by its sender. This field holds the Unix epoch time in nanoseconds as a 64-bits integer value.  |
-
+| `ephemeral` | `Boolean` | optional | This flag indicates the transient nature of the message. |
```

Comment:
Shouldn't we be more specific and clarify that this messages indicates whether the application expects it to be available via the store protoocol?

Reply:
We could, however, there is a link to the WakuMessage rfc which has the explanation of the flag (which includes references to the store protocol).

Reply:
This is a good point, but, for now, as people are still using this as a one-stop API spec, I agree with Franck that something more specific will be clearer. Perhaps "Indicates if message is eligible to be stored by `store` protocol."?

Reply:
Addressed in [cd61aea](https://github.com/vacp2p/rfc/pull/536/commits/cd61aea151f9a3b2682ab5e72cf8ec909d9df877), thanks!